src/component/huskelapp/redigering/huskelapp-redigering.less

```diff
@@ -4,7 +4,24 @@
             60rem,
             100% - 2rem
         ); // max 60rem, men unngå at modalen treff kantane av vindauget på små skjermar
+        .rediger-huskelapp-modal-header {
+            display: flex;
+            align-items: center;
+
+            h2 {
```

Comment:
kan ikke se at vihar noen h2 som har denne cssklassen. Enten må du sette level="2" på linje 169 i huskelapp-redigere-modal.tsx eller så kan du fjerne dette 👍  (eventuelt h3 her, men da ser man hvor sårbart det kan være så da er det bedre kanskje å lage en egen klasse for det isteden for å definere det på html elelemt da det kan endre seg i fremtiden)